### PR TITLE
Add social metadata.

### PR DESCRIPTION
**Pull request by janisz**
Added [OpenGraph](http://ogp.me/) and [Twitter](https://dev.twitter.com/cards/types/summary) metadata to generate cards in social media. Below is an example render of twitter card with link to javers.org

twitter card

### CODE REVIEW

**Comment by bgalek** [comment]
please fix logo size :)

**Comment by bgalek** [comment]
and prefer svg over png

File: _includes/head.html [hunk 1]
```diff
@@ -8,6 +8,13 @@
 <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
 <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
 
+<meta name="twitter:card" content="summary" />
+<meta name="twitter:site" content="@javers_org" />
+<meta property="og:url" content="https://javers.org/" />
+<meta property="og:title" content="{{ page.title }}{{ titleSuffix }}" />
+<meta property="og:description" content="{{ page.description }}" />
+<meta property="og:image" content="https://javers.org/img/logotype.png" />
```
Comment (janisz) [inline _includes/head.html]: According to the documentation this should be a square image (e.g., the same as for favicon). Unfortunately, there is no square logo with dimensions bigger than 144x144.
> A URL to a unique image representing the content of the page. You should not use a generic image such as your website logo, author photo, or other image that spans multiple pages. Images for this Card support an aspect ratio of 1:1 with minimum dimensions of 144x144 or maximum of 4096x4096 pixels. Images must be less than 5MB in size. The image will be cropped to a square on all platforms. JPG, PNG, WEBP and GIF formats are supported. Only the first frame of an animated GIF will be used. SVG is not supported.

Comment (bgalek) [inline _includes/head.html]: I'll upload 144x144 then

Comment (bgalek) [inline _includes/head.html]: @janisz use img/javers-og-image.png

**Comment by janisz** [comment]
@bgalek 

> and prefer svg over png

Please report this on @Twitter support I follow the docs they provided

> **SVG is not supported.**

**Comment by janisz** [comment]
voila_capture 2017-06-25_7-12-8_pm

**Comment by bartoszwalacik** [comment]
thanks @janisz

**Pull request merged**